2022-01-14 11:58:41,272 INFO    MainThread:337302 [wandb_setup.py:_flush():71] setting env: {}
2022-01-14 11:58:41,272 INFO    MainThread:337302 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 11:58:41,272 INFO    MainThread:337302 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-14 11:58:41,273 INFO    MainThread:337302 [wandb_init.py:_log_setup():371] Logging user logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_115841-30fellx8/logs/debug.log
2022-01-14 11:58:41,273 INFO    MainThread:337302 [wandb_init.py:_log_setup():372] Logging internal logs to /home/tbertrand/Bureau/ULM_4channel_pull/wandb/run-20220114_115841-30fellx8/logs/debug-internal.log
2022-01-14 11:58:41,273 INFO    MainThread:337302 [wandb_init.py:init():404] calling init triggers
2022-01-14 11:58:41,273 INFO    MainThread:337302 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-14 11:58:41,274 INFO    MainThread:337302 [wandb_init.py:init():460] starting backend
2022-01-14 11:58:41,274 INFO    MainThread:337302 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-14 11:58:41,281 INFO    MainThread:337302 [backend.py:ensure_launched():216] starting backend process...
2022-01-14 11:58:41,285 INFO    MainThread:337302 [backend.py:ensure_launched():221] started backend process with pid: 337378
2022-01-14 11:58:41,285 INFO    MainThread:337302 [wandb_init.py:init():469] backend started and connected
2022-01-14 11:58:41,287 INFO    MainThread:337302 [wandb_init.py:init():533] updated telemetry
2022-01-14 11:58:41,296 INFO    MainThread:337302 [wandb_init.py:init():563] communicating current version
2022-01-14 11:58:41,655 INFO    MainThread:337302 [wandb_init.py:init():568] got version response 
2022-01-14 11:58:41,655 INFO    MainThread:337302 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-14 11:58:41,817 INFO    MainThread:337302 [wandb_init.py:init():606] starting run threads in backend
2022-01-14 11:58:41,872 INFO    MainThread:337302 [wandb_run.py:_console_start():1810] atexit reg
2022-01-14 11:58:41,873 INFO    MainThread:337302 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-01-14 11:58:41,873 INFO    MainThread:337302 [wandb_run.py:_redirect():1689] Redirecting console.
2022-01-14 11:58:41,874 INFO    MainThread:337302 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-14 11:58:41,874 INFO    MainThread:337302 [wandb_init.py:init():633] run started, returning control to user process
2022-01-14 11:58:42,102 INFO    MainThread:337302 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-14 11:58:42,103 INFO    MainThread:337302 [wandb_run.py:_restore():1752] restore
2022-01-14 11:58:44,384 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 11:58:44,557 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 779
}

2022-01-14 11:58:44,821 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 2460
}

2022-01-14 11:58:44,923 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 779
  total_bytes: 4190
}

2022-01-14 11:58:45,024 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4190
  total_bytes: 4190
}

2022-01-14 11:58:45,125 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4190
  total_bytes: 4190
}

2022-01-14 11:58:45,227 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4190
  total_bytes: 4190
}

2022-01-14 11:58:45,329 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4190
  total_bytes: 4190
}

2022-01-14 11:58:45,433 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4190
  total_bytes: 4190
}

2022-01-14 11:58:45,537 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4190
  total_bytes: 4190
}

2022-01-14 11:58:45,990 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4190
  total_bytes: 4190
}

2022-01-14 11:58:46,370 INFO    MainThread:337302 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4190
  total_bytes: 4190
}
local_info {
}

2022-01-14 11:58:47,448 INFO    MainThread:337302 [wandb_run.py:_append_files():2180] logging synced files
